curriculum/challenges/english/blocks/workshop-accessible-audio-controller/68ed3a96e0ebcc36565b39e5.md

```diff
@@ -0,0 +1,123 @@
+---
+id: 68ed3a96e0ebcc36565b39e5
+title: Step 5
+challengeType: 0
+dashedName: step-5
+---
+
+# --description--
```

Comment:
This would work better as 2 steps instead of 1.

one step for adding the input element with the following attributes
```html
    <input type="range" min="0" max="100" value="50">
```

one step for adding the `aria-labelledby` attribute